index.html

```diff
@@ -16,7 +16,31 @@
   </head>
 
   <body>
-    <h1 class="text-center">Tailwind Cards Challenge</h1><br>
-    <p class="text-center">Check the README for instructions.</p>
+    <section class="flex flex-wrap mx-auto max-w-[500px] md:max-w-[752px] lg:max-w-[1023px]">
```

Comment:
Please avoid using hardcoded widths like 500px, 752px, and 1023px. [Tailwind has utility classes to build responsive designs](https://tailwindcss.com/docs/responsive-design). Please refactor your code accordingly.